gqcp/include/ONVBasis/ONVManipulator.hpp

```diff
@@ -34,7 +34,7 @@ template <typename _DerivedManipulator>
 class ONVManipulator:
     public CRTP<_DerivedManipulator> {
 
-protected:
+public:
```

Comment:
I'm awaiting your comment on issue #668 before I'll move this line into the main `public:` section to avoid a lot of reconstruction work.